resources/views/searches/resources/groups/edit_group.blade.php

```diff
@@ -0,0 +1,94 @@
+
+<!-- Add group modal - OPEN -->
```

Comment:
Should be <!-- Edit group modal - OPEN -->

Reply:
ok